prime-router/metadata/fhir_transforms/senders/SimpleReport/simple-report-sender-transform.yml

```diff
@@ -34,6 +34,15 @@ elements:
     bundleProperty: '%resource.receivedTime'
     value: [ '%resource.collection.collected - 894 seconds' ]
 
+  - name: specimen-received-date-time
```

Comment:
There are two elements in this transformed named specimen-received-date-time

Reply:
Good catch! I've updated them.